content/docs/command-reference/add.md

```diff
@@ -344,3 +342,49 @@ $ tree .dvc/cache
 
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
+
+## Example: Transfer to remote storage
+
+When you have a large dataset in an external location, you may want to add it to
+your project without downloading it to the local file system (for using it
+later/elsewhere). The `--to-remote` option let you skip the download, while
+storing the imported data [remotely](/doc/command-reference/remote). Let's
+initialize a DVC project, and setup a remote:
+
+```dvc
+$ mkdir example # workspace
+$ cd example
+$ git init
+$ dvc init
+$ mkdir /tmp/dvc-storage
+$ dvc remote add myremote /tmp/dvc-storage
+```
+
+Now let's add the `data.xml` to our remote storage from the given remote
+location.
+
+```dvc
+$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml \
+                 --to-remote -r myremote
+...
+```
+
+The only difference that dataset is transferred straight to remote, so DVC won't
+control the remote location you gave but rather continue managing your remote
+storage where the data is now on. The operation will still be resulted with an
+`.dvc` file:
```

Comment:
Possible copy edit here (please don't commit yet)... But I need to make sure I understand `add --to-remote` (see https://github.com/iterative/dvc.org/pull/2172#discussion_r573963049) before finalizing this:

```suggestion
The difference with regular `dvc add` is that, while a `.dvc` file is created,
the data isn't cached. Instead, it's transferred directly to remote storage (as
ig pre-[pushing](/doc/command-reference/push) it).
```

Reply:
OK I updated all my copy edits now (per https://github.com/iterative/dvc.org/pull/2172#discussion_r573978325) 👍

Reply:
Hey @jorgeorpinel, do you need any more updates on the copy-edits after our discussion in the issue?

Reply:
Thanks for the reminder @isidentical please feel free to address the feedback now and then we'll have to make one last review keeping in mind that long discussion we had — hopefully just copy edits that I can commit myself and then approve.

Reply:
> feel free to address the pending feedback

Specifically 2 things are left: https://github.com/iterative/dvc.org/pull/2172#discussion_r578699447 and https://github.com/iterative/dvc.org/pull/2172#issuecomment-776898393 above.